package-lock.json

```diff
@@ -8130,15 +8130,15 @@
         "sha.js": "^2.4.11"
       },
       "engines": {
-        "node": ">=12.13.0 <18.0"
```

Comment:
@trevor-scheer Can we drop explicit limitations on future versions of Node?
AFAIK, it's the only package that does that and it doesn't use a lot of Node API (only `crypto` and `process`).

Reply:
Yeah that's fine with me 👍